server/text_generation_server/layers/attention/cuda.py

```diff
@@ -173,6 +174,41 @@ def paged_attention(
 try:
     import flash_attn_2_cuda
 
+    # try forwarding to see if it works with all dummy inputs
```

Comment:
Wouldn't it be easier to require the minimum needed CUDA capability? We could even skip the import altogether if the hardware doesn't have the right capability.

Reply:
I agree, it would be simpler.

Reply:
yea I agree thats a better method. Updated to use `is_ampere_or_newer = major >= 8 and minor >= 0` in the latest commits and avoid trying the forward pass